### PR TITLE
Update README WEE message for current v5 Shield utils progress

### PR DESCRIPTION
**Pull request by biancadanforth**
@gregglind @motin Changing wording here to reflect our current state/recommendations, as per @pdehaan , we are still getting devs forking the v4 template.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,19 @@
...
-Until support for those workflows become stable, it is still recommended to use this template as it is and create legacy add-ons for your study.
-Chat with us: #shield on Slack about the latest progress and how to help us progress faster away from legacy add-ons.
+In an effort to remove the necessity of creating legacy add-ons for Shield studies, we are working on [supporting a pure Web Extension Experiment workflow in this template](https://github.com/mozilla/shield-studies-addon-template/issues/53) with a new version, v5, of the [Shield utilities](https://github.com/mozilla/shield-studies-addon-utils/). Support for these workflows is not yet stable. In the meantime, **we do not recommend using this v4 template**.
+
+Instead, we recommend that you:
+* build your study as a WEE ([Web Extension Experiment](https://firefox-source-docs.mozilla.org/toolkit/components/extensions/webextensions/index.html))
+* help us test the requisite [experimental Shield API(s)](https://github.com/mozilla/shield-studies-addon-utils/)
+
+Example WEEs using the experimental Shield API(s):
```
Comment (motin) [inline README.md]: "WEEs" -> "Study add-ons" (since web extensions do not use other WEE APIs)

File: README.md [hunk 2]
```diff
@@ -4,11 +4,19 @@
...
+* https://github.com/mozilla/shield-studies-addon-utils/blob/develop/examples/small-study
+* https://github.com/mozilla/shield-cloudstorage
```
Comment (motin) [inline README.md]: Also https://github.com/motin/dataleak-pioneer-shield-study

File: README.md (hunk 1 above)
Comment (biancadanforth) [reply README.md]: Thanks motin for your feedback. This is an abbreviation for Web Extension Experiments, not Web Extensions. I will rephrase anyway.

File: README.md (hunk 2 above)
Comment (biancadanforth) [reply README.md]: That study looks to be an embedded web extension, not a WEE (i.e. I don't see any calls to `browser.study` in `background.js` or a `/privileged` folder.

**Pull request merged**